tests/alfacase/test_case_description.py

```diff
@@ -1374,3 +1374,72 @@ def test_case_description_duplicate_names_between_elements(default_well):
 
     with pytest.raises(InvalidReferenceError, match=re.escape(expected_msg)):
         case.ensure_unique_names()
+
+
+def test_check_fluid_references(default_well: case_description.WallDescription) -> None:
+    """
+    Test _check_fluid_references isn't yielding errors for a correct CaseDescription.
+    """
+
+    known_pvt_properties = set(
+        property
+        for property in vars(case_description.PvtModelsDescription).keys()
+        if "__" not in property
```

Comment:
I see, let's be explicit then:

```suggestion
        if not property.startswith("__") and not property.endswith("___")
```

Reply:
It is good to point out that in python that `__asd` and `__asd__` have two very distinct meanings:

- `__asd`: Is method name that will be mangled, like `_ClassName__asd` to prevent name clashes with other classes in the class hierarchy (parents or children);
- `__asd__`: Is know as dunder methods, python will not mangle then and are protocols, like `__len__` that will be called when the user calls `len(some_obj)` or `__add__` that will be called by `some_obj + other_obj`;

@nicoddemus please let me know if I am forgetting some thing.

Reply:
No, your remarks are on point. 👍 

Just so we are on the same page, this remarks are just added info, and I think we still should update the according to my suggestion above.